### ISSUE TITLE
Error when markdown contain invisible space

### ISSUE THREAD

**Issue by bammoo** [peerigon/markdown-loader#14]
Version: 0.1.7

Console.log info

```
WARNING in ./data/build-markdown/test.md
Module parse failed: /Users/appuser/test/myapp/node_modules/html-loader/index.js!/Users/appuser/test/myapp/node_modules/markdown-loader/index.js!/Users/appuser/test/myapp/data/build-markdown/test.md Unterminated string constant (1:17)
You may need an appropriate loader to handle this file type.
SyntaxError: Unterminated string constant (1:17)
```

I found this invisible space after tried many markdown editor, this character not show in any editor even Sublime Text. I think it was produced by ACE HTML editor.

![http://ww4.sinaimg.cn/large/6110a121gw1f4loqpj4i9j20b708o74r.jpg](http://ww4.sinaimg.cn/large/6110a121gw1f4loqpj4i9j20b708o74r.jpg)

**Comment by theopak** [comment]
I experienced this same error, and it was driving me crazy! I finally found U+2028 LINE SEPARATOR characters in the markdown file I was trying to load, which caused the error. 

The message `SyntaxError: Unterminated string constant (1:17)` in the stack trace refers to a function in line 1 col 17 that parsed the file, AFAICT. It does not indicate a position in the file.

Here's a vim command to find all occurrences of U+2028 in a file and delete them:

    :%s/\%u2028//g
